docs/source/ts-ssdk/validation.rst

```diff
@@ -0,0 +1,137 @@
+.. _TS SSDK validation:
+
+=======================================
+Smithy TypeScript Server SDK Validation
+=======================================
+
+Smithy defines a set of :ref:`constraint traits <constraint-traits>` that can be used to restrict the set of allowed
+values for a shape. These traits do not change the type generated; the type will always correspond with the Smithy type.
+This allows Smithy models to evolve over time, so that model changes will not cause changes that break type checks.
+
+Smithy Server SDKs will only validate input values against constraint traits. While output structures can have
```

Comment:
Should this be "The Smithy Server SDK"?

Reply:
I was trying to (subtly) make a language-agnostic statement here.